DTS-STN/canadian-dental-care-plan
# AB#2975 -- Eligibility intake pages and routes

### Description
Added eligibility pages:
-Type of Application
-Tax Filing
-Date of Birth

Added error pages: 
-Applicant Delegate
-File Your Taxes
-Dob Eligibility

### Related Azure Boards Work Items
AB#2975

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/ad71110a-b86a-4856-a12d-74482c2d018f)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/a492c113-f71a-4ba9-b89c-faef8d901a8d)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/198350cb-2afc-47b2-b704-97251c973d64)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/8644980a-1087-4062-9ebd-aae62964d1f8)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/8461b869-b696-4dc3-ab04-48db280b78da)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/b1f9e08d-4c45-467c-8f02-2ec3fcca9f2f)


### Checklist

### Test Instructions
Run locally and navigate to "/apply" to confirm the correct workflow for eligibility .

### Additional Notes
To test dob use: March - (current day) - 1959
